eigen3/src/Util_TEST.cc

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <ignition/math/eigen3/Util.hh>
+
+using namespace ignition;
+
+/////////////////////////////////////////////////
+/// \brief Test the oriented box converted from a set of vertices
+TEST(EigenUtil, verticesToOrientedBox)
```

Comment:
mmmm the Simple box approach worked ! and gave me the exact box 

I tried that before but on a rotated box in all directions, and it didn't gave me the same box, so that is why I told you that it will not work